### DISCUSSION TITLE
TP-1: Chemical Industry Veteran

### DISCUSSION THREAD

**Discussion by ThePolymerist** [Global-Chem/global-chem#206] [Show and tell]
Hey,

I've spent the last 6 years in chemical product development for global chemical companies working across end market segments such as footwear, industrial adhesives, construction products, and waterproofing. Happy to help answer any questions around commercialization of new chemicals, the fundamentals of the chemical industry, and how to enact real change in a stagnated capital intensive industry.

I write twice a week at my newsletter [The Polymerist
](https://polymerist.substack.com/)

Tony

**Comment by Sulstice** [comment]
Hey @ThePolymerist Could we start gathering resources for common lists used in these products maybe come up with a plan on what lists to add to the network and also how. 

I think that will help us get started.

**Comment by ThePolymerist** [comment]
You just want a list of monomers used to make polymers?

On Tue, Aug 16, 2022 at 2:05 PM Sul ***@***.***> wrote:

> Hey @ThePolymerist <https://github.com/ThePolymerist> Could we start
> gathering resources for common lists used in these products maybe come up
> with a plan on what lists to add to the network and also how.
>
> I think that will help us get started.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Sulstice/global-chem/discussions/206#discussioncomment-3408731>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A2RCB4RAEZSUVH4WO5H3IA3VZPJ5PANCNFSM56SEWQMA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>
-- 
--
Tony Maiorana
The Polymerist <http://polymerist.substack.com/>

**Comment by Sulstice** [comment]
Yeah! I had one that I got from the IUPAC polymer book, reference is in the README but here is an example.

https://github.com/Sulstice/global-chem/blob/development/global_chem/global_chem/materials/polymers/common_monomer_repeating_units.py

I think if we had more of these it could be worthwhile to get an idea of the functional group space of polymers. What do you think?

**Comment by ThePolymerist** [comment]
It might be tough because I usually only know the common names.

What exactly is the goal? Structure property relationships of polymers
based on monomer composition?
--
Tony Maiorana
The Polymerist <http://polymerist.substack.com/>

**Comment by Sulstice** [comment]
I guess, I have no idea initially of what the chemical makeup of a lot of polymers are. For an easy start, if we begin at chemical space we can get an idea of what kind of functional groups do you guys work with holistically. 

I'm also wondering what functional groups are common between different communities (or rather regions of chemical space).

Does that make sense? 

Common names I think are better than IUPAC names @LierB has been doing a conversion of any stringent IUPAC stuff into more preferred names for the rest of the lists.
